flow-model-generator/modules/flow-model-generator-core/src/main/java/io/ballerina/flowmodelgenerator/core/model/Property.java

```diff
@@ -95,9 +95,9 @@ public <T> T valueAsType(TypeToken<T> typeToken) {
     public static final String DATA_TYPE_KEY = "type";
     public static final String DATA_TYPE_DOC = "Type of the variable";
 
-    public static final String CHECK_ERROR_LABEL = "Check Flag";
+    public static final String CHECK_ERROR_LABEL = "Check Error";
```

Comment:
What if we use `Return On Error`? @hasithaa

Reply:
We discussed to use `Check Error` for now. Let's see other's feedback and change it later as needed.